schema/resolvers.js

```diff
@@ -249,8 +260,8 @@ const Resolvers = {
   },
 
   RoutingCondition: {
-    routingValue: ({ id }) => {
-      return { conditionId: id };
+    routingValue: ({ id, answerId }) => {
+      return { conditionId: id, answerId };
```

Comment:
Could we return the whole condition here? So that the field resolvers would just be picking them off the existing object?